src/core/backend/pearl/holder.rs

```diff
@@ -58,6 +64,20 @@ impl Holder {
         self.start_timestamp <= timestamp && timestamp < self.end_timestamp
     }
 
+    pub(crate) fn is_outdated(&self) -> bool {
```

Comment:
I think it's better to close not recently used blobs. It should be easy to track last time blob was written.

Reply:
As well we can set not to close too small blobs